sunpy/net/dataretriever/sources/goes.py

```diff
@@ -97,6 +97,6 @@ def _can_handle_query(cls, *query):
         chkattr =  ['Time', 'Instrument']
         chklist =  [x.__class__.__name__ in chkattr for x in query]
         for x in query:
-            if x.__class__.__name__ == 'Instrument' and x.value == 'goes':
+            if (x.__class__.__name__ == 'Instrument' and x.value == 'goes'):
```

Comment:
Yeah, I added them when I added something else and then removed that but forgot the brackets. 
I'll fix.